middleware/swagger-validator.js

```diff
@@ -298,7 +298,11 @@ var wrapEnd = function (req, res, next) {
         mHelpers.debugError(err, debug);
       }
 
-      return next(err);
+      if (responseValidatorHook) {
```

Comment:
How does inserting the hook here help more than just chaining to another Express middleware?  Isn't `next(err)` essentially performing the same function?